epics.go

```diff
@@ -69,6 +69,7 @@ type Epic struct {
 	Downvotes               int         `json:"downvotes"`
 	UserNotesCount          int         `json:"user_notes_count"`
 	URL                     string      `json:"url"`
+	Confidential		bool	    `json:"confidential"`
```

Comment:
Please place the field between `State` and `WebURL` following the ordering in the docs...

Reply:
Yes, done!